{"name": "test_with_issue_link", "status": "passed", "start": 1588906491720, "stop": 1588906491720, "uuid": "d7322ecb-d96e-47c2-8b2c-39b9e90058b0", "historyId": "e21b7c934dee72527417c5f147526411", "testCaseId": "e099b2fe0344ed9be1f2c29527e05911", "fullName": "test_link#test_with_issue_link", "labels": [{"name": "suite", "value": "test_link"}, {"name": "host", "value": "pinrongdeMacBook-Pro.local"}, {"name": "thread", "value": "90385-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "test_link"}], "links": [{"type": "issue", "url": "140", "name": "Pytest-flaky test retries shows like test steps"}]}